content/01-motivation.md

```diff
@@ -81,56 +79,94 @@ Why are we not comparing directly all digits with the expected result?
 
 ---
 
-## Tests help developers of your code
+## Tests help other developers
 
-- Tests make it possible to refactor the code
-- Code is unsustainable without runnable tests and becomes legacy software
+- Tests make it easier to **refactor the code** (rewrite code while keeping functionality)
+- Code can become unsustainable without runnable tests and becomes legacy software
 - Documentation which is up to date by definition
 - Easier for external developers to contribute to the project without breaking your code
+  (**you may immediately see problems in your code but others may not**)
 
 Suiting up to modify untested code:
 
-<img src="{{ site.baseurl }}/img/suit.jpg" style="width: 400px;"/>
+```{image} img/suit.jpg
+:alt: Suiting up to modify untested code
+:width: 300px
+```
```

Comment:
````suggestion
```{figure} img/suit.jpg
:alt: Old-fashioned diving suit
:width: 300px

Suiting up to modify untested code.
```
````

We can have separate image captions like this.

Reply:
Thanks!